src/library_fs.js

```diff
@@ -1640,6 +1641,15 @@ mergeInto(LibraryManager.library, {
           return lazyArray.chunks[chunkNum];
         });
 
+
+        if (usesGzip||!datalength){
+            //if the server uses gzip or doesn't supply the length, we have to download the whole file to get the (uncompressed) length
```

Comment:
space after //`, and indentation should be 2 spaces (i see some of the code you are replacing had wrong formatting, let's fix this while you replace the code)